plugin.py

```diff
@@ -16,7 +16,19 @@ class LspVuePlugin(NpmClientHandler):
     server_directory = 'server'
     server_binary_path = os.path.join(server_directory, 'node_modules', 'vue-language-server', 'bin', 'vls')
 
-    def on_client_configuration_ready(self, configuration: dict):
+    @classmethod
+    def configuration(cls) -> sublime.Settings:
+        settings = super().configuration()
+        options = settings.get('initializationOptions', {})
+        cls._add_formatting_options(options)
+        settings.set('initializationOptions', options)
+        return settings
```

Comment:
I thought we were going to avoid doing changes specific to v2 version in individual packages.

This can be easily avoided by calling `on_client_configuration_ready` from `configuration` in new API handler.
The method can be if-checked even to make it error-proof.

Reply:
Sounds good.